.azure-pipelines.yml

```diff
@@ -16,26 +17,32 @@ jobs:
         python.version: '3.6'
       Python37:
         python.version: '3.7'
-    maxParallel: 4
+    maxParallel: 3
 
   steps:
+    # Specify to use the python version defined above
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+
+    # Additional info about the build
   - script: |
-      # Additional info about the build
     - uname -a
     - free -m
     - df -h
     - ulimit -a
 
   # Setup python environment
-  steps:
   - script: |
     # Create test environment for package
     - conda env create -n test python=$PYTHON_VER -f devtools/conda-envs/openff.yaml
     - source activate test
-
     # Build and install package
-  - python setup.py develop --no-deps
+    - python setup.py develop --no-deps
+    displayname: "Conda Env Creation"
 
+  # Display MongoDB, Conda Environment, Python version
```

Comment:
We need to spin up a MongoDB, I don't think this is done?

Reply:
Its not. But I think we have to get the YAML working once first and then we can work from there.

Reply:
And by working, I mean syntax correct, not everything passing correct.